jsx/Form.js

```diff
@@ -1572,6 +1573,7 @@ ButtonElement.propTypes = {
   name: PropTypes.string,
   label: PropTypes.string,
   type: PropTypes.string,
+  disabled: PropTypes.bool,
```

Comment:
should we add default false below if not prop not provided?

Reply:
@zaliqarosli I think if no property is supplied there won't be a disabled added to the button.